test/ecto/adapters/postgres_test.exs

```diff
@@ -618,6 +618,17 @@ defmodule Ecto.Adapters.PostgresTest do
     assert all(query) ==
              ~s{SELECT s0."x" FROM "schema" AS s0 ORDER BY s0."x" ASC NULLS FIRST, s0."y" DESC NULLS FIRST}
 
+    order_by =
+     [asc: Ecto.Query.dynamic([], exists(from other_schema in "schema", where: other_schema.x == parent_as(:r).x, select: [other_schema.x]))]
```

Comment:
We should also have an example without dynamic.

Reply:
Okay so I actually ended up removing the dynamic tests, because it seemed to me that the dynamic/not-dynamic divide is now tested in the ecto PR. Happy to add them back in here though if you disagree 👍